megalinter/reporters/UpdatedSourcesReporter.py

```diff
@@ -63,6 +64,11 @@ def produce_report(self):
                 f" in folder {updated_sources_dir}.\n"
                 "Download it from artifacts then copy-paste it in your local repo to apply linters updates"
             )
+
+            repo = git.Repo(os.path.realpath(self.github_workspace))
```

Comment:
Do that only if APPLY_FIXES = all , or matches the linter :)

- self.master will give you the Linter instance
- self.master.master will give you the MegaLinter instance
- so self.master.master.github_workspace will work :)

Also, check if you are on GitHub: if you are, don't make git operation, as they are already defined in the next GitHub Action :) ( if it works, we can deprecate it in a next major version)

Reply:
how to check the git platform being used?

Reply:
if GITHUB_REPOSITORY is set, you're on GitHub actions :)

Reply:
*think* I've catered for both of those now? 🤞